doc/source/how-to/documenting.rst

```diff
@@ -93,62 +99,62 @@ figure. The following RST files provide examples of how to create ``index.rst``
 
-                section_A/index
-                section_B/index
+                section-A/intro
+                section-B/intro
 
-    .. tab-item:: section_A/index.rst
+    .. tab-item:: section-A/intro.rst
 
         .. code-block:: rst
 
             Section A
             #########
 
-            This is the content of the `section_A/index.rst` file.
+            This is the content of the `section-A/intro.rst` file.
 
             .. toctree::
 
-                section_1
-                section_2
+                section-1
+                section-2
                 ...
 
-    .. tab-item:: section_B/index.rst
+    .. tab-item:: section-B/intro.rst
 
         .. code-block:: rst
 
             Section B
             #########
 
-            This is the content of the `section_B/index.rst` file.
+            This is the content of the `section-B/intro.rst` file.
 
             .. toctree::
 
-                another_section
+                another-section
                 ...
 
 While you do not include the ``.rst`` extension when defining the section
 structure in the ``toctree`` directive, the index file referenced for each
-section should be named ``index.rst``.
+section should have a short descriptive name.
 
 After you build documentation locally as described in :ref:`Build
-documentation`, the first-level heading in the ``index.rst`` file for each
+documentation`, the first-level heading in the index file for each
 section is shown as a clickable link in the title bar of the
 library's HTML documentation. For more information on defining the
 documentation structure, see `Getting Started
 <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ in the Sphinx
 documentation.
 
-Indicating RST titles
-+++++++++++++++++++++
+Titles and headings in RST files
+++++++++++++++++++++++++++++++++
 
-Within RST files, heading titles are to use sentence case per the
-`capitalization guidelines <https://developers.google.com/style/capitalization>`_
-in the *Google developer documentation style guide*. The line that follows
-the heading title must have a string of characters that is the same length
-as the heading title. If the length of the characters under the heading title
-is less than the length of the heading title, Sphinx generates a warning.
+As indicated in `Capitalization <https://developers.google.com/style/capitalization>`_
+in the *Google developer documentation style guide*, titles and headings are to use
+sentence case. In RST files, the line that follows a title or heading must have a
+string of characters that is the same length as the heading or title. If the length
+of the characters is less than the length of the title or heading, Sphinx generates
+a warning.
 
-For consistency within PyAnsys libraries, the use of the following special characters
-is recommended for headings but is not enforced:
+For consistency within PyAnsys libraries, the use of these special characters
+is recommended but not enforced:
```

Comment:
I virtually always see ``===`` used as the special characters for section-level headings. Should we consider bumping it u and dropping this bullet: "For paragraph-level headings, use ``+++``." Given that the right navigation pane isn't being used to show more than 3 levels, for a paragraph-level heading, I use bold text. I doubt anyone would even understand what "paragraph-level heading" even means.